README.md

```diff
@@ -2,3 +2,30 @@ FuseSoC standard core library
 =============================
 
 This is the standard core library to be used with [FuseSoC](https://github.com/olofk/fusesoc)
+
+### Installation
+
+Clone this repository and add it to your `fusesoc.conf`.  For example
+in `~/.config/fusesoc/fusesoc.conf`:
+
+```
+[main]
+cores_root =
+ /home/joe/work/fusesoc-cores
+cache_root = /home/joe/work/fuse-cache
+build_root = /home/joe/work/fuse-builds
+```
+
+### Contributing
+
+Cores for fusesoc should follow the following guidelines.
+
+ - **No Code** Please do not store code in `fusesoc-cores`.  Please store
+   your code in a separate repo or somehwere it can be fetch with the `url`
+   provider.
+ - If your core is really small please consider storing in the
+   [tiny-cores](http://github.com/fusesoc/tiny-cores) repo.
+ - **Stay Modern** Use the modern sections in your core file like `fileset`
+   and `parameter`, please no obsolete `verilog` sections.
+ - **Use a Version** For cores using the `github` provider please try to used
```

Comment:
For cores without a version, use version 0 and point to a location which is unique for that revision (e.g. the sha for git repos or revision for svn repos instead of pointing to master, which might change over time). If you need to refer to a newer version of the upstream repo, still without a proper version, step the revision number (0-r1, 0-r2..etc)

Reply:
OK, Ill add that.  I guess there are two options here, "versioned" and "pseudo-version" I'll add that.  Do you think there is really a good case for the 0-r1.. version?  Why would we not be able to give a version to a core?  I was thinking to just give a v1.0 to the wb_sdram_ctrl core.  But I guess thats not my call as the original author, so maybe that is the case.

Reply:
Many upstream repos lack explicit versions (@cliffordwolf I'm looking at you, picorv32 ;) ) and we don't want to make up our own versioning scheme for cores that we don't own. For wb_sdram_ctrl I think you should be perfectly fine to slap on a tag unless @skristiansson is dead set against it, as I consider both you and me somewhat as co-maintainers at this point.

Pseudo-version is a good name for it